### DISCUSSION TITLE
Running the pick_place exercise

### DISCUSSION THREAD

**Discussion by minhnh** [JdeRobot/RoboticsAcademy#1586] [Q&A] [answered]
I recently tried to follow the [tutorial for the pick and place exercise](https://jderobot.github.io/RoboticsAcademy/exercises/IndustrialRobots/pick_place), but the exercise doesn't seem to be there after I start the docker instance (as seen in the attached screenshots). This is true for both the latest image and the 2.4.2 release shown in the tutorial.

Are there some additional steps necessary for me to run this exercise locally? Do I need a local installation of ROS and/or gazebo?

![20220316-RoboticsAcademy_local-latest](https://user-images.githubusercontent.com/1168534/159244109-991f67ac-fee9-40e4-a077-61f51a691b23.png)
![20220316-RoboticsAcademy_local-2 4 2](https://user-images.githubusercontent.com/1168534/159244121-05b27d9e-c7fa-440b-862e-12c90983fd88.png)

**Comment by jmplaza** [comment, marked as answer]
HI @minhnh, "Pick and place exercise" is outdated. I'm sorry... It requires improvement to get it running with current RADI...
